src/server/requestHandlers.js

```diff
@@ -111,50 +109,40 @@ fluid.defaults("sjrk.storyTelling.server.getStoryHandler", {
  *
  * @param {Object} request - a Kettle request that includes an ID for the story to retrieve
  * @param {Component} dataSource - an instance of sjrk.storyTelling.server.dataSource.couch.story
- * @param {String} uploadedFilesHandlerPath - the path on the server to uploaded files
  */
-sjrk.storyTelling.server.handleGetStory = function (request, dataSource, uploadedFilesHandlerPath) {
+sjrk.storyTelling.server.handleGetStory = function (request, dataSource) {
     var id = request.req.params.id;
     var promise = dataSource.get({directStoryId: id});
 
-    promise.then(function (response) {
+    var noAccessErrorMessage = "An error occurred while retrieving the requested story";
```

Comment:
We may need to review error messages and make them more configurable on the server. Perhaps a JIRA for the future.

Reply:
I agree, and they should be localizable. This goes for the logging and messages in the rest of the request handlers and middleware, as well. I have filed [SJRK-419](https://issues.fluidproject.org/browse/SJRK-419) to tackle this work.